Tasks/PackerBuildV0/Tests/L0Linux.ts

```diff
@@ -54,23 +54,23 @@ let a: any = <any>{
             "code": 0,
             "stdout": "{ \"some-key\": \"some-value\" }"
         },
-        "packer validate -var resource_group=testrg -var storage_account=teststorage -var image_publisher=Canonical -var image_offer=UbuntuServer -var image_sku=14.04.4-LTS -var location=South India -var capture_name_prefix=Release-1 -var skip_clean=true -var script_relative_path=dir3/somedir/deploy.sh -var package_path=/tmp/dir1/somedir/dir2 -var package_name=dir2 -var script_arguments=\"subdir 1\" false -var subscription_id=sId -var client_id=spId -var client_secret=spKey -var tenant_id=tenant -var object_id= /tmp/tempdir/100/default.linux.template-fixed.json": {
```

Comment:
You should modify test to ensure that these parameters are present in var file. I think test mocks Write-File to write to console. You can grep console output to ensure that this content is correct.
In this task, passing correct parameters is the most important aspect. Hence we need to test them thoroughly.

Reply:
added 2 tests for this. first one verifies for default template, the other one for custom template